### PR TITLE
[Home Header] Improvements "TV Shows" default

### PR DESCRIPTION
**Pull request by Nanomani**
Count "TV Show" added :  

before 

after

### CODE REVIEW

File: 1080i/Includes_Furniture.xml [hunk 1]
```diff
@@ -3079,7 +3079,7 @@
...
         <value condition="!Library.HasContent(Music)">[COLOR=$VAR[ColorHighlight]]$INFO[Window(home).Property(Movies.Count)][/COLOR] $LOCALIZE[20342]  •  [COLOR=$VAR[ColorHighlight]]$INFO[Window(home).Property(Episodes.Count)][/COLOR] $LOCALIZE[20360]  •  [COLOR=$VAR[ColorHighlight]]$INFO[Pvr.BackendChannels][/COLOR] $LOCALIZE[19019]</value>
         <value condition="String.IsEmpty(Pvr.BackendChannels)">[COLOR=$VAR[ColorHighlight]]$INFO[Window(home).Property(Movies.Count)][/COLOR] $LOCALIZE[20342]  •  [COLOR=$VAR[ColorHighlight]]$INFO[Window(home).Property(Episodes.Count)][/COLOR] $LOCALIZE[20360]  •  [COLOR=$VAR[ColorHighlight]]$INFO[Window(home).Property(Music.SongsCount)][/COLOR] $LOCALIZE[134]</value>
-        <value>[COLOR=$VAR[ColorHighlight]]$INFO[Window(home).Property(Movies.Count)][/COLOR] $LOCALIZE[20342]  •  [COLOR=$VAR[ColorHighlight]]$INFO[Window(home).Property(Episodes.Count)][/COLOR] $LOCALIZE[20360]  •  [COLOR=$VAR[ColorHighlight]]$INFO[Window(home).Property(Music.SongsCount)][/COLOR] $LOCALIZE[134]  •  [COLOR=$VAR[ColorHighlight]]$INFO[Pvr.BackendChannels][/COLOR] $LOCALIZE[19019]</value>
```
Comment (beatmasterRS) [inline 1080i/Includes_Furniture.xml:3082]: A few changes need to be made:

- please don't replace the episodes count (3082)
- a wrong localized string is used for tvshows (3070)

**Review by beatmasterRS — CHANGES_REQUESTED** [review]

**Comment by Nanomani** [comment]
I hope it is good this time ...

**Comment by Nanomani** [comment]
I have try to add "TV shows" playlist with your advise [https://forum.kodi.tv/showthread.php?tid=337862&pid=3053004#pid3053004](url)

This seems simple. :)

But with :  
`<content>plugin://script.embuary.helper/?info=getpathstats&amp;path=$INFO[Container(300).ListItem.Property(path)]&amp;prefix=PlaylistStats</content>`

(I have use a Label just after this code. So I m sure that this code is read.)

the result of this :
```
Window(Home).Property(PlaylistStats_TVShowCount)
Window(Home).Property(PlaylistStats_Episodes)
Window(Home).Property(PlaylistStats_UnwatchedEpisodes)
```
is empty. 

Anyway it is probably in my side. Nothing urgent here. 
You have other thing to do. It is only for sharing ...
I will keep looking ...

**Comment by beatmasterRS** [comment]
Looks good, thank you :)

**Pull request merged**